extensions/amp-carousel/amp-carousel.md

```diff
@@ -57,11 +57,15 @@ If present, the user may advance past the first item or the final item.
 
 If present, advances the slide to the next slide without user interaction.
 By default it will advance a slide in 5000 millisecond intervals (5 seconds)
-but will use the value of the `autoplay` attribute if present (minimum of 1000 ms).
-Value of `autoplay` must be numeric representation in milliseconds, ex. `autoplay=5000`.
+and can be overridden by the `delay` attribute.
 If `autoplay` is present it will also attach the `loop` attribute to
 `amp-carousel` if `loop` is not already present.
 
+**delay** (type=slides only)
+By default a slide will will advance a slide in 5000 millisecond intervals (5 seconds)
```

Comment:
1. "will will" -> "will"
2. "By default a slide will advance ... when `autoplay` is specified"